elytron/src/main/java/org/wildfly/extension/elytron/ElytronDefinition.java

```diff
@@ -108,6 +108,11 @@
  */
 class ElytronDefinition extends SimpleResourceDefinition {
 
+    /**
+     * System property which if set to {@code true} will cause the JVM wide default {@link SSLContext} to be restored when the subsystem shuts down.
+     */
+    public static final String RESTORE_DEFAULT_SSL_CONTEXT = ElytronDefinition.class.getPackage().getName() + ".restore-default-ssl-context";
```

Comment:
Can this be package protected and the javadoc note that it's for internal test use only?

The package protected is just me being anal since the whole class is.